proposed/MPID Messaging.md

```diff
@@ -396,47 +384,221 @@ impl MpidMessage {
 }
 ```
 
-To send an MPID Message, a client would do something like:
-
-```rust
-let mpid_message = MpidMessage::new(my_mpid: Mpid, recipient: ::routing::Authority::Client,
-                                    metadata: Vec<u8>, body: Vec<u8>);
-
-```
-
 Account types held by MpidManagers
 
 ```rust
-struct Outbox {
+struct OutboxAccount {
     pub sender: ::routing::NameType,
+    pub sender_clients: Vec<::routing::Authority::Client>,
```

Comment:
OK.  I've got that in the flowchart as a PutFailure - i.e. I was expecting to have to keep the PutRequest at Managers(A) until we heard back from Managers(B).

I can update the flowchart with this change if required, but we need to handle the inbox the same as the outbox as far as Online messages go (which may be required eventually anyway I suppose) - so we have to also handle PostFailure for these messages to any Client(A)s and remove them from the vector of authorities.

Also, we'll be sending responses for Put requests as Post messages, which is probably confusing for Clients, and further, we'll be sending these responses to all online instances of the same client, even though it should probably just go to the instance which created the MpidMessage.